### PR TITLE
Update child info url to dependent

### PR DESCRIPTION
**Pull request by jbalboni**
Step 6 in the connected issue. This updates the url from child-information to dependent-information and adds a migration.

### CODE REVIEW

File: src/js/hca/config/migrations.js [hunk 1]
```diff
@@ -50,5 +50,17 @@ export default [
     }
 
     return { formData: newData, metadata };
+  },
+  // 2 -> 3, we're updating the url for the dependent info page since it's for dependents
+  // and not just children anymore
+  ({ formData, metadata }) => {
+    const url = metadata.returnUrl || metadata.return_url;
```
Comment (cehsu) [inline src/js/hca/config/migrations.js]: Would `metadata` ever contain the camelcase version of these two, should only the snake case version be used here? The camelcase version is available on formData I believe, but I'm not sure it is available on metadata.

File: test/hca/config/migrations.unit.spec.js [hunk 2]
```diff
@@ -88,4 +88,35 @@ describe('HCA migrations', () => {
...
   });
+  describe('third migration', () => {
+    const migration = migrations[2];
+    it('should update url when it matches', () => {
+      const data = {
+        formData: {
+          'view:reportChildren': false
+        },
+        metadata: {
+          returnUrl: '/household-information/child-information'
```
Comment (cehsu) [inline test/hca/config/migrations.unit.spec.js]: Same questions as above.

File: src/js/hca/config/migrations.js [hunk 3]
```diff
@@ -50,5 +50,17 @@ export default [
...
+    let newMetadata = metadata;
+
+    if (url === '/household-information/child-information') {
+      newMetadata = _.set('returnUrl', '/household-information/dependent-information', metadata);
```
Comment (cehsu) [inline src/js/hca/config/migrations.js]: Answer to above question will influence how this `returnUrl` or `return_url` key is named.

**Review by cehsu — APPROVED** [review]
LGTM. Had a question about whether camelcase should be used.

File: src/js/hca/config/migrations.js (hunk 1 above)
Comment (jbalboni) [reply src/js/hca/config/migrations.js]: The vast majority of the time it should be camel case. The front end only ever reads and writes the return url in camel case. I believe the backend will send it in snake case when prefilling data, which the front end ignores in general, but I think we have to account for in migrations.

**Pull request merged**